man/systemd-udevd.service.xml

```diff
@@ -160,6 +175,15 @@
           terminated due to kernel drivers taking too long to initialize.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>udev.timeout_signal=</varname></term>
+        <term><varname>rd.udev.timeout_signal=</varname></term>
+        <listitem>
+          <para>Specifies a signal that <filename>systemd-udevd</filename> will send to
+          worker on timeouts. Note that kernel command line option overrides both,
```

Comment:
"to workers on timeout"

Not sure how to fix it, but the grammar is somehow wrong here. Maybe "Note that kernel command line option overrides both the setting in the configuration file and the one on the program command line."